packages/strapi-parts/src/Link/Link.stories.mdx

```diff
@@ -21,6 +21,17 @@ Links are used to send users to another page or an anchor. They are used for ext
 - Links text should be concise and describe their purpose. Users need to know exactly what they are about to do.
 - Use capital letters.
 
+
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/Link)
+
+## Imports
+
+```js
+import { Link } from '@strapi/parts/Link';
```

Comment:
The `to` props is shown as a func in the ArgsTable can you customise it so it displays a string?